plugin-server/tests/helpers/clickhouse.ts

```diff
@@ -54,5 +54,6 @@ export async function delayUntilEventIngested<T extends any[] | number>(
         }
         await delay(delayMs)
     }
+    expect(1).toBeNull() // Delay exhausted, try re-running the test
```

Comment:
while we can check the count in the end, I'd argue that this makes it a lot simpler for folks who aren't familiar with these tests, when they see a test failure due to this.